src/h2olog/h2olog.cc

```diff
@@ -228,11 +228,7 @@ int main(int argc, char **argv)
 {
     std::unique_ptr<h2o_tracer> tracer;
     if (argc > 1 && strcmp(argv[1], "quic") == 0) {
-        fprintf(stderr, "warning: 'quic' mode is deprecated. Use 'raw' mode instead.\n");
-        tracer.reset(create_raw_tracer());
-        --argc;
-        ++argv;
-    } else if (argc > 1 && strcmp(argv[1], "raw") == 0) {
+        fprintf(stderr, "warning: 'quic' mode is deprecated.\n");
```

Comment:
Can we remove the concept of "mode" altogether, entirely relying on `getopt` to determine the necessary action?